Cargo.toml

```diff
@@ -31,7 +31,9 @@ x25519 = [ "x25519-dalek" ]
 random = [ "getrandom" ]
 aes = [ "aes-gcm", "cipher" ]
 blake2b = [ "blake2", "digest" ]
-curl-p = [ "bee-ternary" ]
+sponge = []
+curl-p = [ "bee-ternary", "sponge" ]
+kerl = [ "bee-ternary", "sponge", "byteorder", "lazy_static", "tiny-keccak" ]
```

Comment:
didn't we want to group these into a feature known as "deprecated_do_not_use" or something?

Reply:
Renamed